lib/services/elasticsearch.js

```diff
@@ -332,6 +333,10 @@ class ElasticSearch extends Service {
     assertNoRouting(esRequest);
     assertWellFormedRefresh(this.config.apiVersion, esRequest);
 
+    if (!this.kuzzle.indexCache.exists(esRequest.index, esRequest.type)) {
+      return Bluebird.reject(new PreconditionError(`Index '${esRequest.index}' and/or collection '${esRequest.type}' don't exist`))
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ASemicolon)